xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -1863,7 +1863,7 @@ absl::Status IrEmitterUnnested::EmitCollectivePermute(
   const int64_t partition_count = hlo_config.num_partitions();
 
   auto operands = instr->operands();
-  std::vector<CollectiveThunk::Buffer> buffers;
+  std::vector<NcclCollectiveThunk::Buffer> buffers;
```

Comment:
`NcclCollectiveThunk` has been renamed to just `CollectiveThunk` as of https://github.com/openxla/xla/commit/822cd3b8534652724a51d199328585cd225827b6. Could you please help fix? Thank you!

Reply:
Huh, I have no idea where these changes came from. I probably messed up with git somewhere. Looking.

Reply:
Fixed. Thanks!